app/main.py

```diff
@@ -1 +1,24 @@
-# write your code here
+from app.errors import (NotVaccinatedError,
+                        OutdatedVaccineError,
+                        NotWearingMaskError)
+
+
+def go_to_cafe(friends: list, cafe: str) -> str:
+    vaccinated_friends = 0
```

Comment:
why do you need this logic? If a friend is not vaccinated, it's indeed an error condition, and returning an error message